tests/app/Http/Controllers/Api/V1_0/ProjectControllerTest.php

```diff
@@ -104,7 +104,10 @@ public function testShowPublicSuccessfully()
         $this->json('get',
             '/api/projects/' . $project->id,
             [],
-            ['X-VMS-API-Key' => $this->getApiKey()]
+            [
+                'X-VMS-API-Key' => $this->getApiKey(),
+                'Authorization' => 'Bearer ' . JWTAuth::fromUser($volunteer),
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class 'JWTAuth' in method 'testShowPublicSuccessfully'.](https://www.codacy.com/app/ym-huang0808/VMS/file/2452055397/issues/source?bid=2972826&fileBranchId=3248926#l109)